hudi-hadoop-common/src/test/java/org/apache/hudi/metadata/TestHoodieTableMetadataUtil.java

```diff
@@ -273,7 +281,8 @@ public void testGetLogFileColumnRangeMetadata() throws Exception {
             metaClient,
             columnsToIndex,
             Option.of(HoodieTestDataGenerator.AVRO_SCHEMA_WITH_METADATA_FIELDS),
-            HoodieMetadataConfig.MAX_READER_BUFFER_SIZE_PROP.defaultValue());
+            HoodieMetadataConfig.MAX_READER_BUFFER_SIZE_PROP.defaultValue(),
+            metadataConfig.isOptimizedLogBlocksScanEnabled());
```

Comment:
We need to update this to not use metadata config

Reply:
Since there is no write config present for this test, and the parent class does not provide one, will just use the default value of the config.